contentpal-testing/src/main/java/org/dmfs/android/contentpal/testing/predicates/BackRefArgument.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.android.contentpal.testing.predicates;
+
+import android.support.annotation.NonNull;
+
+import org.dmfs.android.contentpal.Predicate;
+import org.dmfs.optional.Optional;
+
+
+/**
+ * A simple constant argument with no back reference.
+ *
+ * @author Marten Gajda
+ */
+public final class BackRefArgument implements Predicate.Argument
```

Comment:
This class in not used at the moment, did you miss to use it in the test or is it for later?

Reply:
Oh, right. I guess you can remove that unless you think it might be useful later on.

Reply:
Ok, I've removed it.